packages/shared/src/UI/components/SettingsBoard/GasSection.tsx

```diff
@@ -72,27 +93,11 @@ export function GasSection(props: GasSectionProps) {
                                     ? GAS_OPTION_NAMES[gasOptionType]
                                     : t.gas_settings_custom()}
                             </span>
-                            {activeTab === GasSettingsType.Basic ? (
-                                <span className={classes.price}>
-                                    {' '}
-                                    {new BigNumber(
-                                        (gasOptions?.[gasOptionType].suggestedMaxFeePerGas as string | undefined) ?? 0,
-                                    ).toFixed(2)}
-                                    {' Gwei'}
-                                </span>
-                            ) : (
-                                <span className={classes.price}>
-                                    {' '}
-                                    {transactionOptions
-                                        ? formatWeiToGwei(
-                                              ((transactionOptions as Transaction).maxFeePerGas as
-                                                  | string
-                                                  | undefined) ?? 0,
-                                          ).toFixed(2)
-                                        : 0}
-                                    {' Gwei'}
-                                </span>
-                            )}
+                            <span className={classes.price}>
+                                {' '}
+                                {customPrice}
+                                {' Gwei'}
```

Comment:
```suggestion
                                {` ${customPrice} Gwei`}
```

Reply:
Hmmm, we disallowed template strings in JSX.